source/ways-of-working/index.html.md.erb

```diff
@@ -9,4 +9,5 @@ weight: 3
 
 - [Principles](principles/index.html)
 - [Concepts and standards](standards/index.html)
-- [Starting a new component](new-component/index.html)
\ No newline at end of file
+- [Starting a new component](new-component/index.html)
+- [Troubleshooting](troubleshooting/index.html)
```

Comment:
Does this belong in ways of working or a new top level section ?

Reply:
I think its ways of working ?

Reply:
I think possibly a new top level section to catch all references of potential troubleshooting